src/data/news/coverage.yml

```diff
@@ -365,6 +410,7 @@
   Params:
     sortDate: "2019-03-13"
     icon: "dcrInMedia.svg"
+    outlet: "WSJ"
```

Comment:
Spell out Wall Street Journal?

Reply:
Updated. I think I pulled "WSJ" from the logo.

Some [contexts](https://en.wikipedia.org/wiki/The_Wall_Street_Journal) are also adding "The" while some don't. I avoid leading "The" in my data because it screws up sorting. Can change to anything though.